pkgs/by-name/ha/handheld-daemon/package.nix

```diff
@@ -1,35 +1,35 @@
 {
+  lib,
+  python3Packages,
   fetchFromGitHub,
+
+  # dependencies
   hidapi,
-  kmod,
-  lib,
-  python3,
   toybox,
-  lsof
 }:
-python3.pkgs.buildPythonApplication rec {
+python3Packages.buildPythonApplication rec {
   pname = "handheld-daemon";
-  version = "3.6.1";
+  version = "3.6.2";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "hhd-dev";
     repo = "hhd";
     rev = "refs/tags/v${version}";
-    hash = "sha256-IdpSRb66G+WzTv/BC29r2OjO1b4VdWbV6OSzOoiFAO0=";
+    hash = "sha256-W1Ap6yTryBDozKe3aO413Fu0RBul9kEA9ACUTdYyOKM=";
   };
 
-  propagatedBuildInputs = with python3.pkgs; [
+  build-system = with python3Packages; [
+    setuptools
+  ];
+
+  dependencies = with python3Packages; [
     evdev
-    hidapi
-    kmod
+    pyserial
     pyyaml
     rich
     setuptools
-    toybox
     xlib
-    pyserial
-    lsof
```

Comment:
Please do not remove kmod and lsof from the dependencies, handheld-daemon uses them [here](https://github.com/hhd-dev/hhd/blob/c664e1530301a893648e6b09b6d1bc7b41568197/src/hhd/controller/physical/imu.py#L445) and [here](https://github.com/hhd-dev/hhd/blob/c664e1530301a893648e6b09b6d1bc7b41568197/src/hhd/plugins/overlay/x11.py#L136) respectively

Reply:
I also see:

- `toybox "id"`
- `systemd "udevadm"`
- `systemd "systemd-inhibit"`
- `dbus "dbus-monitor"`
- `efibootmgr`

You can see why I'm nervous this approach might break something.